docs/installation.rst

```diff
@@ -13,12 +13,27 @@ Requirements
 - The ``--container-runtime`` option must correspond to the containerization tool you use.
 - ``ansible-builder`` version ``3.x`` requires Python ``3.9`` or higher.
 
+.. note::
+
+   The **recommended** approach to install ``ansible-builder`` is using the
+   ``ansible-dev-tools`` package.
+
+   `Ansible Development Tools (ADT) <https://ansible.readthedocs.io/projects/dev-tools/>`_
+   aims to streamline the setup and usage of several tools needed in order to
+   create `Ansible <https://www.ansible.com>`_ content. ADT combines critical Ansible
+   development packages into a unified Python package.
```

Comment:
```suggestion
   `Ansible Development Tools (ADT) <https://ansible.readthedocs.io/projects/dev-tools/>`_ is a single Python package that contains all the tools you need to create high quality Ansible content.
   The ADT package helps you quickly set up a development environment, generate new collections, and then build and test to ensure consistent and robust automation.
```

Reply:
I'd get more to the point and tell the reader what ADT is from the start. Also address the reader more directly through the 2nd person singular ("you").

Rather than ADT on its own as an acronym, I'd suggest "the ADT package" to avoid ambiguity.